.github/workflows/_reusable-eng-tools-test.yaml

```diff
@@ -45,12 +45,19 @@ jobs:
             eng
             ${{ inputs.sparse-checkout-paths }}
 
+      # TODO: Merge latest from main/RPSaaSMaster after checkout
+
       - name: Use Node ${{ matrix.node-version }}.x
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}.x
 
-      - run: npm install --no-package-lock
+      - if: github.ref_name == 'typespec-next' || github.base_ref == 'typespec-next'
```

Comment:
Would this preclude us from creating PRs into RPSaaSMaster from typespec-next.  unfamiliar with when ref_name or base_ref might match this in a PR.

Reply:
Good question.  `ref_name` should be the short name of the source branch (always available), and `base_ref` should be the name of the target branch (if the build is a PR).  So I think this condition should only be "true" for builds of branch `typespec-next` itself, or PRs targeting `typespec-next`.  If you created a PR from `typespec-next` to `RPSaaSMaster`, I think `ref_name` would be `<pr_number>/merge`, and `base_ref` would be `RPSaaSMaster`.